tests/test_simple_app.cc

```diff
@@ -12,10 +12,10 @@ void ReqHandle(const SimpleData& req, SimpleApp* app) {
 
 int main(int argc, char *argv[]) {
   int n = 100;
-  SimpleApp app(0);
+  SimpleApp app(0, 0);
   app.set_request_handle(ReqHandle);
 
-  Start();
+  Start(0);
```

Comment:
since it is always `0` for `ps::Start()`, do we have to add arg `customer_id` for it?

Reply:
it's just a test file, in the following PR (against mxnet, this will be different numbers)